pages/linux/xclock.md

```diff
@@ -0,0 +1,23 @@
+# xclock
+
+> Display the time in analog or digital form.
+
+- Display an analog clock:
+
+`xclock`
+
+- Display a 24 hour digital clock with the hours and minutes fields only:
+
+`xclock -digital -brief`
+
+- Display a digital clock using an strftime format string (see strftime(3)):
+
+`xclock -digital -strftime {{format}}`
+
+- Display a 24 hour digital clock with the hours, minutes and seconds fields that updates every second:
```

Comment:
Perhaps this will read better?
```suggestion
- Display a 24 hour digital clock with the hour, minute and second fields that updates every second:
```